tests/lib/devicetree/api/src/main.c

```diff
@@ -2667,6 +2667,70 @@ ZTEST(devicetree_api, test_mbox)
 				  DT_NODELABEL(test_mbox_zero_cell)), "");
 }
 
+ZTEST(devicetree_api, test_memory_attr)
+{
+	#define REGION_RAM_ATTR		(0xDEDE)
+	#define REGION_RAM_NOCACHE_ATTR	(0xCACA)
+
+	#define TEST_FUNC(p_name, p_base, p_size, p_attr)	\
+		{ .name = (p_name),				\
+		  .base = (p_base),				\
+		  .size = (p_size),				\
+		  .attr = (p_attr),				\
+		}
```

Comment:
hmm how does this work w/o a `,`?

Reply:
The comma is added by https://github.com/zephyrproject-rtos/zephyr/blob/07be64e431148ebe5c79fdd019b6650b0e2ef897/include/zephyr/devicetree/memory-attr.h#L29-L35